src/GitHub.InlineReviews/Services/PullRequestStatusBarManager.cs

```diff
@@ -112,42 +121,39 @@ StatusBar FindSccStatusBar(Window mainWindow)
             return contentControl?.Content as StatusBar;
         }
 
-        class RaisePullRequestCommand : ICommand
+        class UsageTrackingCommand : ICommand
```

Comment:
I changed it to use the `IShowCurrentPullRequestCommand` directly rather than using `dte.Commands.Raise`. The problme was I lost the metrics.

I'm thinking it would be a better idea to record the metrics in the command itself? Only issue is, you wouldn't be able to differentiate where the user found the command (not that it matters yet in this case).

Reply:
I'll fix this in another PR. It's working for the moment.

Reply:
I've moved the tracking of `NumberOfShowCurrentPullRequest` into the command itself (where I think it belongs). `RaisePullRequestCommand` is now obsolete.